### DISCUSSION TITLE
CS2 account data like XP lvl, PR lvl, Medals, Commends, Wins

### DISCUSSION THREAD

**Discussion by t-gajjar** [ianlucas/cs2-inventory-simulator#145] [General]
Want to create a localhost to maintain CS2 data for multiple accounts.
Data like XP lvl, PR lvl, Medals, Commends, Wins

Any help to get these data ?

**Comment by ianlucas** [comment]
Not sure what you mean by that, can you be more specific on your goal? The app currently only handles player inventories.

**Comment by crashzk** [comment]
I think that's probably what I've been thinking about recently, @ianlucas.

Are you planning on further enhancing Inventory Simulator's features? Not just an inventory, I imagine a full-fledged website, something similar to the CS2 interface.

![image](https://github.com/user-attachments/assets/52eff32b-f17f-4d2f-b155-8a9a7852d71b)

User profile containing information such as their level within our servers, their kill statistics, etc. Ranking integrated into the website, support for third-party ranking plugins from CSSharp or even HLstatsX:CE 1.6.19 (2025): https://github.com/SnipeZilla/hlxce

In other words, it would be the main website where users connect via Steam, as it is now, but in addition to the inventory, it would have more information about rankings on the website and user profiles.

Something similar to what we have with Flute CMS: https://github.com/Flute-CMS/cms
